lib/Bio/EnsEMBL/Tark/Hive/RunnableDB/HGNCNames.pm

```diff
@@ -0,0 +1,113 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+=head NAME
+
+  Bio::EnsEMBL::Tark::Hive::PipeConfig::HGNCNames
+
+
+=head1 DESCRIPTION
+
+  Runnable for the loading of the Tark db from a Core db.
+
+
+=head1 DESCRIPTION
+
+  A pipeline for loading ensembl genomic features into the Tark DB. Requires
+  connection parameters for the Tark and Core dbs, as well was matching tags to
+  describe the entries.
+
+  This process can be run over a whole core db or by passing a list of gene_ids
+  in a comma separated string, thus allowing multiple instances to handle the
+  loading of the Tark db.
+
+=cut
```

Comment:
I think the Description text is carried over. Please update the text as relevant to this class.

Reply:
Fixed in  https://github.com/Ensembl/tark-loader/pull/8